packages/interface/src/components/index.js

```diff
@@ -1,8 +1,13 @@
+export { default as ActionItem } from './action-item';
 export { default as ComplementaryArea } from './complementary-area';
 export { default as ComplementaryAreaMoreMenuItem } from './complementary-area-more-menu-item';
 export { default as FullscreenMode } from './fullscreen-mode';
 export { default as InterfaceSkeleton } from './interface-skeleton';
-export { default as PinnedItems } from './pinned-items';
 export { default as MoreMenuDropdown } from './more-menu-dropdown';
 export { default as MoreMenuFeatureToggle } from './more-menu-feature-toggle';
-export { default as ActionItem } from './action-item';
+export { default as PinnedItems } from './pinned-items';
+export { default as PreferencesModal } from './preferences-modal';
+export { default as PreferencesModalTabs } from './preferences-modal/tabs';
```

Comment:
Are tabs, sections, and toggles specific to the modal, or can they be used in any place?

Reply:
They're specific to the modal, there's quite a bit of CSS to make it appear the way it does.